vector/v.edit/select.c

```diff
@@ -259,6 +260,7 @@ int sel_by_cat(struct Map_info *Map, struct cat_list *cl_orig, int layer,
     }
 
     Vect_destroy_list(List_tmp1);
+    Vect_destroy_cat_list(cl);
```

Comment:
We cannot unconditionally free `cl`. It may have been set from `cl_orig`, in which case it is not to be freed here.